app/assets/stylesheets/custom.css.scss

```diff
@@ -57,6 +57,16 @@ input[type="submit"] {
   box-shadow: none;
 }
 
+#small_submit_button {
```

Comment:
Avoid using id selectors<br>Selector `small_submit_button` should be written in lowercase with hyphens